### ISSUE TITLE
Tracking issue: new Bevy book

### ISSUE THREAD

**Issue by alice-i-cecile** [bevyengine/bevy-website#158]
Following-up on https://github.com/bevyengine/rfcs/pull/23,  here are the steps that I see we need to get this implemented:

- [ ] review and merge https://github.com/bevyengine/bevy/pull/2094 for a shiny new breakout game example
- [ ] create a skeleton for the new book structure (#154).
- [ ] reorganize the  Learn section into Quick Starts / Book
- [ ] write each chapter of the book
- [ ] write the quick start guide for the Breakout game

If you want a chapter, claim it here :) Otherwise I'll slowly write them up and come to the community for help as needed once #154 is merged.

**Comment by alice-i-cecile** [comment]
Closing in favor of #265.
